examples/check-downloaded-file-name-and-content/README.MD

```diff
@@ -0,0 +1,21 @@
+# Check the Name and the Contents of a Downloaded File
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to obtain the name and the content of a file downloaded during a test.
+
+During the test, a link click starts a file download. To verify download success and obtain the name and the content of the file, a [RequestLogger](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/) is attached to the test with the [t.requestHooks](https://devexpress.github.io/testcafe/documentation/reference/test-api/test/requesthooks.html) method.
+
+To verify success of a download, a response status code is checked with the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) method.
```

Comment:
Not only status code. File name and file content are checked also.

What about this: 

To verify the downloaded file information the [RequestLog.contains](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/contains.html) method is used in combination with [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) method.

Reply:
> The `RequestLog.contains` method in combination with `t.expect.ok` method verify the downloaded file information.